test/performance/performance-tests.sh

```diff
@@ -69,6 +69,8 @@ $(echo "${dev_config}")"
   echo ">> Updating benchmark $1"
   ko delete -f "${benchmark_path}"/${TEST_CONFIG_VARIANT}
   ko apply -f "${benchmark_path}"/${TEST_CONFIG_VARIANT} || abort "failed to apply benchmark $1"
+  kubectl apply configmap -n "${TEST_NAMESPACE}" config-mako --from-file="${benchmark_path}/prod.config" || abort "failed to apply config-mako configmap"
```

Comment:
This command will get an error - 

I think we'll have to replace with the following two commands:
```
kubectl delete configmap config-mako -n "${TEST_NAMESPACE}" --ignore-not-found=true
kubectl create configmap config-mako -n "${TEST_NAMESPACE}" --from-file="${benchmark_path}/prod.config" || abort "failed to create config-mako configmap"
```